### PR TITLE
added password support

### PR DESCRIPTION
**Pull request by venadHD**

### CODE REVIEW

**Comment by PHLAK** [comment]
Thank you for the pull request, it means a lot to me when others are willing to invest their time into making Directory Lister better.  However, I will not be merging this PR as password protected listings is outside of the scope of Directory Lister, [as I mentioned previously](https://github.com/DirectoryLister/DirectoryLister/issues/131#issuecomment-252432347).

That being said there's a lot that could be done to improve this PR.  For example, please use a single pull request for all changes pertaining to a single logical feature (i.e. adding password protection) instead of submitting multiple pull requests ([132](https://github.com/DirectoryLister/DirectoryLister/pull/132), [133](https://github.com/DirectoryLister/DirectoryLister/pull/133), [134](https://github.com/DirectoryLister/DirectoryLister/pull/134)).  Additionally, providing a detailed summary of the proposed changes you're making and why you're making them helps the maintainer(s) (in this case me) understand those changes.  Lastly, I'll be reviewing your submission in more detail and making some comments on the code directly with the intention of providing feedback you can use going forward to help you improve.

Thanks again for your contributions and please don't let this stop you from making future contributions to Directory Lister or any other project for that matter.

File: resources/default.config.php [hunk 1]
```diff
@@ -8,6 +8,9 @@
     'list_sort_order'           => 'natcasesort',
     'theme_name'                => 'bootstrap',
     'external_links_new_window' => true,
+    
+    'protected_by_password'		=> true,
+	'password'					=> "123456",
```
Comment (PHLAK) [inline resources/default.config.php]: Please try and keep formatting consistent when adding to / modifying a file.

**Comment by venadHD** [comment]
Hmmm, how to make multiple file editions to single PR? When i edit it i
only have option to merge

**Comment by PHLAK** [comment]
You have to use the [git](https://git-scm.com/) command line client or [GitHub Desktop](https://desktop.github.com) client.

It sounds like you don't have much experience with git, if you'd like to learn more please check out the [Try Git](https://try.github.io/) tutorial or for a more in-depth overview check out the [Pro Git Book](https://git-scm.com/book/en/v2).

**Comment by venadHD** [comment]
Very thanks for reply with my ignorance about github :P, i will setup it.

Thanks for reply

**Pull request closed without merging**